content/en/docs/helm/helm_install.md

```diff
@@ -50,6 +50,8 @@ set for a key called 'foo', the 'newbar' value would take precedence:
 To check the generated manifests of a release without installing the chart,
 the '--debug' and '--dry-run' flags can be combined.
 
+CRD manifests in the `crds/` directory can't be validated by `--dry-run` flag. See [Helm's CRDs caveats](docs/chart_best_practices/custom_resource_definitions.md#some-caveats-and-explanations) for more details.
```

Comment:
Can you prepend this with: **Note: **. Check https://github.com/helm/helm-www/pull/1185#discussion_r702816234 again for details.

Reply:
Done @hickeyma